stubs/Pillow/PIL/Image.pyi

```diff
@@ -36,21 +36,14 @@ _Color: TypeAlias = (
 class _Writeable(SupportsWrite[bytes], Protocol):
     def seek(self, __offset: int) -> Any: ...
 
-NORMAL: Literal[0]  # deprecated
-SEQUENCE: Literal[1]  # deprecated
-CONTAINER: Literal[2]  # deprecated
-
 class DecompressionBombWarning(RuntimeWarning): ...
 class DecompressionBombError(Exception): ...
 
-MAX_IMAGE_PIXELS: int | None
-USE_CFFI_ACCESS: Incomplete
+MAX_IMAGE_PIXELS: Final[int]
```

Comment:
I'm not sure this is correct. According to https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.open 

> This threshold can be changed by setting [MAX_IMAGE_PIXELS](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.MAX_IMAGE_PIXELS). It can be disabled by setting Image.MAX_IMAGE_PIXELS = None.

I can't see any mention of this in https://pillow.readthedocs.io/en/stable/releasenotes/10.0.0.html either.

(Noticed in https://github.com/matrix-org/synapse/pull/15898)